From 3239f6d6798d86fdbf8e03aa741d9990bcfffe79 Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@php.net>
Date: Thu, 31 Aug 2017 07:55:45 +0200
Subject: [PATCH] move AC_DEFINE in the right position (in bundled lib
 conditon)

---
 ext/sqlite3/config0.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/sqlite3/config0.m4 b/ext/sqlite3/config0.m4
index 0e5fbc5..78fd622 100644
--- a/ext/sqlite3/config0.m4
+++ b/ext/sqlite3/config0.m4
@@ -85,11 +85,11 @@ if test $PHP_SQLITE3 != "no"; then
       other_flags="$other_flags -DSQLITE_ENABLE_ICU=1"
     fi
 
+    AC_DEFINE(HAVE_SQLITE3_ERRSTR, 1, [have sqlite3_errstr function])
     PHP_SQLITE3_CFLAGS="-I@ext_srcdir@/libsqlite $other_flags $threadsafe_flags $debug_flags"
     PHP_INSTALL_HEADERS([ext/sqlite3/libsqlite/sqlite3.h])
   fi
 
-  AC_DEFINE(HAVE_SQLITE3_ERRSTR, 1, [have sqlite3_errstr function])
   AC_DEFINE(HAVE_SQLITE3,1,[ ])
 
   sqlite3_sources="sqlite3.c $sqlite3_extra_sources"
-- 
2.1.4

